src/spryker/README.md

```diff
@@ -26,4 +26,7 @@ attribute('app.services'):
   - jenkins-runner
 ``` 
 
+To generate and encrypt secrets for pipeline environments: 
+https://github.com/inviqa/k8s-project-cluster/blob/0.1.x-dev/docs/cluster-management/sealed-secrets/README.md
```

Comment:
This is a private repo at the moment

Reply:
Is it ok if I change this link later when that doc is uploaded somewhere publicly? or should we not put this link here?